core/os/os_linux.odin

```diff
@@ -265,16 +291,27 @@ foreign libc {
 	@(link_name="write")            _unix_write         :: proc(fd: Handle, buf: rawptr, size: c.size_t) -> c.ssize_t ---;
 	@(link_name="lseek64")          _unix_seek          :: proc(fd: Handle, offset: i64, whence: c.int) -> i64 ---;
 	@(link_name="gettid")           _unix_gettid        :: proc() -> u64 ---;
+	@(link_name="getuid")           getuid              :: proc() -> uid_t ---;
 	@(link_name="getpagesize")      _unix_getpagesize   :: proc() -> c.int ---;
 	@(link_name="stat64")           _unix_stat          :: proc(path: cstring, stat: ^Stat) -> c.int ---;
 	@(link_name="fstat")            _unix_fstat         :: proc(fd: Handle, stat: ^Stat) -> c.int ---;
 	@(link_name="access")           _unix_access        :: proc(path: cstring, mask: c.int) -> c.int ---;
 
+	@(link_name="getpwnam")         _unix_getpwnam      :: proc(name: cstring) -> ^passwd ---;
+	@(link_name="getpwuid")         getpwuid            :: proc(uid: uid_t) -> ^passwd ---;
```

Comment:
Similar here.

Reply:
Same reason as getuid. There didn't need to be a wrapper for getpwuid. I was mostly following the style of the pre-existing stuff in here, which is also why I didn't use temp allocator, since most of the functions didn't use that.